openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSOSMReader.java

```diff
@@ -412,5 +415,15 @@ protected void finishedReading() {
 		procCntx.finish();
 	}
 
-
+	@Override
+	protected double getElevation(ReaderNode node) {
+		if (processElevationFromPreprocessedData) {
+			double ele = node.getEle();
+			if (Double.isNaN(ele)) { // this should npt happen if preprocessing worked correctly
```

Comment:
Typo in comment: npt -> not
Also if this should not happen, we should at least log a warning.

Reply:
fixed.